torch/csrc/jit/runtime/profiling_graph_executor_impl.cpp

```diff
@@ -442,6 +461,11 @@ void ProfilingGraphExecutorImpl::runProfilingOptimizations(
     return;
   }
 
+  if (!exceptionsEnabled()) {
```

Comment:
if this needs to be on the profiling path, maybe a small comment what this pass is for.